saleor/graphql/webhook/subscription_query.py

```diff
@@ -48,22 +78,44 @@ def validate_query(self) -> list[Union[GraphQLSyntaxError, ValidationError]]:
             self.error_code = WebhookErrorCode.GRAPHQL_ERROR.value
             return errors
 
+        subscription = self._get_subscription(self.ast)
+        if not subscription:
+            self.error_code = WebhookErrorCode.MISSING_SUBSCRIPTION.value
+            return [
+                ValidationError(
+                    message="Subscription operation can't be found.",
+                    code=WebhookErrorCode.MISSING_SUBSCRIPTION.value,
+                )
+            ]
+
+        field_names = [
+            selection.name.value for selection in subscription.selection_set.selections
+        ]
+        if len(field_names) > 1 and set(field_names) != {"event"}:
```

Comment:
Can we extract this if with some meaningful name to increase readability? Maybe it's just me, but it took me a little to understand the idea here.